src/about.html

```diff
@@ -39,186 +39,207 @@
+					(<a href="https://twitter.com/hugoroyd" target="_blank" rel="nofollow"><strong>@hugoroyd</strong></a>) started the legal analysis and brought the project to life. Since then, more people have joined the team and have contributed through the reviewer community. Ultimately, all the work is transparent and the discussions happen in public. Our work is <a href="thanks.html" target="_blank">funded</a> by non-profits organisations and individual donations and gets released as <a href="http://fsfe.org/about/basics/freesoftware.html">free software</a> and open data.</a>
 				</p>
 				<p>
-					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html">our classification</a>.
+					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html" target="_blank">our classification</a>.
 				</p>
 				<p>
 					<strong>What does “ToS;DR” mean?</strong> the name is inspired by internet acronym TL;DR which stands for "Too Long; Didn't Read" and is often used on blogs and emails when a block of text is just really long and that people are too lazy to read the whole stuff. It was intended more as a code name than as a real name. But it seems that people like it (do you?)
 				</p>
 				<p>
-					<strong><a href="contribute.html" class="btn btn-big btn-primary">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
+					<strong><a href="contribute.html" class="btn btn-big btn-primary" target="_blank">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
 				</p>
 			</section>
 		</div><!--hero-unit-->
 
 		<div class="container" id="tosdr-content" role="main">
 			<section id="contact">
 				<div class="page-header">
-					<h2 id="contact">Contact us <a href="http://twitter.com/tosdr" class="btn btn-big btn-info">Follow @tosdr on Twitter</a></h2>
+					<h2 id="contact">Contact us <a href="https://twitter.com/tosdr" target="_blank" rel="nofollow" class="btn btn-big btn-info">@tosdr</a> on Twitter</h2>
 				</div>
 				<style type="text/css">.zzz, .zzz a { float:right; font-size:.001px; color:transparent; display:inline-block; width:0px; text-decoration:none; }
 				</style>
 				<p>
-					You can reach us directly at <a href="https://forum.tosdr.org">our reviewers forum</a>. You can also contact us by email to:
+					You can reach us directly at <a href="https://forum.tosdr.org" target="_blank" style="display:block;text-align:center;text-decoration:none;color:#fff;background-image:linear-gradient(270deg,#159,#389,#37c);width:8.5em;border-radius:16px;padding:2px 8px;">Our reviewers forum</a>. You can also contact us by email at
```

Comment:
```suggestion
					You can reach us directly at <a href="https://forum.tosdr.org" target="_blank">Our reviewers forum</a>. You can also contact us by email at
```

That does not look really good and breaks the new lines. The link is already blue

![grafik](https://user-images.githubusercontent.com/40178814/97676984-cef9f580-1a91-11eb-885e-492c0ebc5348.png)

Normal is wayyy better 

![grafik](https://user-images.githubusercontent.com/40178814/97677024-e1742f00-1a91-11eb-9e15-d01d81ea07ec.png)